MapboxMobileEvents/Categories/NSUserDefaults+MMEConfiguration.m

```diff
@@ -290,20 +290,30 @@ - (NSString *)mme_userAgentString {
             NSBundle.mme_mainBundle.mme_bundleVersionString,
             NSBundle.mme_mainBundle.bundleIdentifier,
             NSBundle.mme_mainBundle.infoDictionary[(id)kCFBundleVersionKey]];
```

Comment:
Non-blocker for the time being.

In case you want to limit code duplication and the potential for missing a fallback value, you might implement this as a NSBundle category.

Note:
```
@implementation NSBundle (MapboxMobileEvents)

/*! Bundle Short version (eg. "1.0.2")
@property (nonatomic, readonly) NSString *shortVersion;

/*! The version number of the bundle. (Build number which can be very large eg. "1439") */
@property (nonatomic, readonly) NSString *version;

/*! The human-readable name of the bundle. */
@property (nonatomic, readonly) NSString *bundleName;

@end
```

Then a matching implementation which provides these default values so we don't need to duplicate between these top/bottom sections.

This approach would then be able to provide testable expectations by leveraging a mock bundle which provides values you might check for problematic values such as nil, or special characters.

Reply:
If we were using them in more than one place I might consider adding the abstraction, since this is the only spot we look at these properties I'd rather have all the logic in-line in a single function.

Because Obj-C has no namespaces, we prefix all our Category methods with `mme_` to be safe and not contribute to the `first` problem of dynamically extending class namespaces